pkgs/applications/office/qownnotes/default.nix

```diff
@@ -38,8 +39,20 @@ stdenv.mkDerivation {
     qtsvg
     qtwebsockets
     qt5compat
+    botan2
   ] ++ lib.optionals stdenv.isLinux [ qtwayland ];
 
+  postPatch = ''
+    substituteInPlace ./libraries/botan/botan.pri \
+      --replace "PKGCONFIG += botan-2" ""
```

Comment:
What is the reason for not using pkgconfig？

Reply:
botan-2 was never found. I didn't manage to get around that. Any suggestions?

Reply:
You need add `pkg-config` to `nativeBuildInputs`

Reply:
Thank you!
Unfortunately, I get a lot of `undefined reference` errors on other parts of the code when I use `pkg-config`. 😬

Reply:
I did not reproduce this problem when testing，can you give more details

Reply:
Oh, how silly of me. I was playing around directly in https://github.com/pbek/QOwnNotes/blob/main/default.nix and my direnv from the `shell.nix` (which was Qt5) seems to conflict with Qt6 from the nix-build command! It really works, thank you very much!